pkg/network/nettop/main.go

```diff
@@ -6,24 +6,34 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/network"
-	"github.com/DataDog/datadog-agent/pkg/network/config"
+	networkConfig "github.com/DataDog/datadog-agent/pkg/network/config"
 	"github.com/DataDog/datadog-agent/pkg/network/tracer"
 )
 
 func main() {
+	cfgpath := flag.String("config", "/etc/datadog-agent/datadog.yaml", "The Datadog main configuration file path")
+	flag.Parse()
+
 	if supported, err := tracer.IsTracerSupportedByOS(nil); !supported {
 		fmt.Fprintf(os.Stderr, "system-probe is not supported: %s\n", err)
 		os.Exit(1)
 	}
 
-	cfg := config.New()
+	config.Datadog.SetConfigFile(*cfgpath)
+	if _, err := config.Load(); err != nil {
```

Comment:
Unless the Datadog main configuration file is loaded, the local variable [detectedFeatures](https://github.com/DataDog/datadog-agent/blob/0c98c9c0254999cc19164b2ec949870c537725cf/pkg/config/environment_detection.go#LL40C2-L40C18) in `pkg/config` is `nil`. Then, the function [IsFeaturePresent](https://github.com/DataDog/datadog-agent/blob/0c98c9c0254999cc19164b2ec949870c537725cf/pkg/config/environment_detection.go#L60-L74), which is invoked in [fentry.LoadTracer](https://github.com/DataDog/datadog-agent/blob/0c98c9c0254999cc19164b2ec949870c537725cf/pkg/network/tracer/connection/fentry/tracer.go#L30-L34), causes a panic.